climpred/bootstrap.py

```diff
@@ -393,6 +394,99 @@ def varweighted_mean_period_threshold(control, sig=95, iterations=500, time_dim=
     )
 
 
+def _bootstrap_hindcast_over_init_dim(
+    hind,
+    hist,
+    verif,
+    dim,
+    resample_dim,
+    iterations,
+    metric,
+    comparison,
+    alignment,
+    compute,
+    reference_compute,
+    resample_uninit,
+    reference_alignment,
+    **metric_kwargs,
+):
+    """Helper function for looping bootstrap skill.
+    Needed for hindcast when resampling over init dimension."""
+    pers_skill = []
+    bootstrapped_init_skill = []
+    bootstrapped_uninit_skill = []
+    # reset inits when probabilistic, otherwise tests fail
+    for i in range(iterations):
+        # resample with replacement
+        smp_hind = _resample(hind, resample_dim)
```

Comment:
Up to you, but maybe we could have `resample_dim='init'` in the signature to this function to make it more readable that this is over `init`. Or just have this be fixed without needing the argument at all. It's clear from the function name and docstring, but just a thought.